frontend/modelarchive/src/main/java/com/amazonaws/ml/mms/archive/ModelArchive.java

```diff
@@ -78,7 +78,7 @@ public static ModelArchive downloadModel(String modelStore, String url)
             }
         }
 
-        if (url.startsWith(".")) {
+        if (url.contains("..")) {
```

Comment:
What is this used for? Is this a "relative paths" check?

Reply:
If the url contains .., hacker can navigate to any file path.